chapter05/qli/ex2.pl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+
+chomp( my @inputs = <STDIN> );
+
+print "1234567890"x5, "\n" ;
```

Comment:
Thanks very much @dpratyusha . Somehow Github doesn't auto refresh with the new comments. I found it in your code as well. Nice one! :+1: